roles/bootstrap-os/tasks/bootstrap-centos.yml

```diff
@@ -15,13 +15,4 @@
   when: fastestmirror.stat.exists
 
 - name: Install packages requirements for bootstrap
-  action:
-    module: "{{ ansible_pkg_mgr }}"
-    name: "{{ item }}"
-    state: latest
-  register: bs_pkgs_task_result
-  until: bs_pkgs_task_result|succeeded
-  retries: 4
-  delay: "{{ retry_stagger | random + 3 }}"
-  with_items: "libselinux-python"
-
+  raw: yum -y install libselinux-python
```

Comment:
You should use the [yum module](http://docs.ansible.com/ansible/yum_module.html).